### PR TITLE
Bug report changes

### PR DESCRIPTION
**Pull request by pazos**
Requires https://github.com/koreader/android-luajit-launcher/pull/309

How the new message looks:

## Devices with a crash.log
with

## Devices without a crash.log
without

## Android 
android-with

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/7709)

### CODE REVIEW

**Comment by poire-z** [comment]
> remove version and device id from the "report a bug message"

Why ? It's quite useful if a user can know this by just looking at it again.

**Comment by pazos** [comment]
> > remove version and device id from the "report a bug message"
> 
> Why ? It's quite useful if a user can know this by just looking at it again.

For no good reason, I suppose :/. Just seemed redundant info (it's in the crash.log and in the about section). Can add them again, no problem :)

**Comment by poire-z** [comment]
But the OS Android version + Device name (my phone technical name) is only available in the Report a bug InfoMessage (I just discovered that a few days ago - and found it nice to have :)

**Comment by NiLuJe** [comment]
Yeah, I get the reasoning in that it's technically redundant in the log (... provided the log is not truncated by the user), but mildly useful in the UI ;).

TL;DR: +1 for keeping it ;).

**Comment by Frenzie** [comment]
Definitely keep, keep, keep please!

**Comment by pazos** [comment]
Yup. Thanks for the feedback.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ normally be found in the KOReader directory:
 * `.adds/koreader/` directory for Kobo
 * `applications/koreader/` directory for Pocketbook
 
-Android won't have a crash.log file because Google restricts what apps can log, so you'll need to obtain logs using `adb logcat KOReader:I ActivityManager:* AndroidRuntime:* DEBUG:* *:F`.
 
+Android logs are kept on memory. Please go to Help->Bug Report to dump these logs to a file
```
Comment (Frenzie) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]:
```suggestion
Android logs are kept in memory. Please go to Help → Bug Report to save these logs to a file.
```

File: frontend/ui/elements/common_info_menu_table.lua [hunk 2]
```diff
@@ -53,10 +53,26 @@ common_info.about = {
 common_info.report_bug = {
     text = _("Report a bug"),
     keep_menu_open = true,
-    callback = function()
+    callback_func = function()
+        local DataStorage = require("datastorage")
+        local log_path = string.format("%s/%s", DataStorage:getDataDir(), "crash.log")
+        local common_msg = T(_("Please report bugs to \nhttps://github.com/koreader/koreader/issues\n\nVersion:\n%1\n\nDetected device:\n%2"),
+            Version:getCurrentRevision(), Device:info())
+        local log_msg = T(_("Attach %1 on your bug report"), log_path)
```
Comment (Frenzie) [inline frontend/ui/elements/common_info_menu_table.lua:61]:
```suggestion
        local log_msg = T(_("Attach %1 to your bug report."), log_path)
```
